lib/ex_doc/markdown/earmark.ex

```diff
@@ -74,6 +76,46 @@ defmodule ExDoc.Markdown.Earmark do
     "$$\n#{content}\n$$"
   end
 
+  # Convert admonition blockquotes to sections for screen reader accessibility
+  defp fixup(
+         {"blockquote", blockquote_attrs, [{tag, h_attrs, h_content, h_meta} | rest] = ast,
+          blockquote_meta}
+       )
+       when tag in ["h3", "h4"] do
+    {h_classes, h_attrs} = Enum.split_with(h_attrs, &match?({"class", _}, &1))
```

Comment:
You can also use List.keytake and similar. :) Similar below.

Reply:
Ok, I'm glad to hear that 😅
I think I can make it look better with keytake/keystore

Reply:
If you take, you probably don't need keystore, you can just prepend it: `[{"class", val} | attrs]`